helm/zot/values.schema.json

```
@@ -384,7 +395,7 @@
                     "type": "object"
                 },
                 "clusterIP": {
-                    "type": ["null","string"]
+                    "type": "null"
```

I feel this should be reverted.

```suggestion
                    "type": ["null","string"]
```